### PR TITLE
MMCA-4478 Add unit tests for views on historic statement frontend

### PR DESCRIPTION
**Pull request by kuldeepwadhwa9**
Description - Added unit tests to meet the statement and branch coverage to 92%

Below has been done as part of this PR

Added tests
Minor refactoring to remove warnings
Remove components, repositories, views, models, helpers from scoverage.coverageExcluded files
Footerlinks, laguageSelection, phaseBanner, siteHeader, summaryListOld files deleted because they were not used anywhere in the project

### CODE REVIEW

File: test/models/ModeSpec.scala [hunk 1]
```diff
@@ -14,16 +14,15 @@
  * limitations under the License.
  */
 
-package views.helpers
+package models
 
-import play.api.i18n.Messages
+import base.SpecBase
 
-object PageTitle {
```
Comment (AdamCoops) [inline test/models/ModeSpec.scala:21]: This is used by the layout.scala.html
Will this not effect functionality by deleting it?

File: test/views/CheckYourAnswersViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        implicit val viewDoc: Document = view(DutyDefermentStatement, Some(true))
+        titleShouldBeCorrect(viewDoc,
+          "cf.historic.document.request.review.DutyDefermentStatement.title")
+
+        shouldContainSecondaryHeading(view = viewDoc, accountNumberText =
+          msg("cf.account.detail.requested.deferment-account-secondary-heading.NiAccount"))
+      }
+    }
+
+    trait Setup {
```
Comment (AdamCoops) [inline test/views/CheckYourAnswersViewSpec.scala:59]: We typically put the Setup last in order to be consistent.

File: test/views/components/SpanSpec.scala [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      }
+
+      "visually hidden is set to false" in new Setup {
+        implicit val spanView: Document = view(visuallyHidden = false)
+        shouldNotDisplayVisuallyHiddenKey()
+        shouldDisplaySpanWithMessageKey()
+      }
+    }
+  }
+
```
Comment (AdamCoops) [inline test/views/components/SpanSpec.scala:40]: You have an extra line here

File: test/models/ModeSpec.scala (hunk 1 above)
Comment (kuldeepwadhwa9) [reply test/models/ModeSpec.scala:21]: pageTitle is an argument in layout.scala.html
i've checked it before deleting and it was alright.

File: test/views/CheckYourAnswersViewSpec.scala (hunk 2 above)
Comment (kuldeepwadhwa9) [reply test/views/CheckYourAnswersViewSpec.scala:59]: I've now moved them for all the specs i've worked on.

**Pull request merged**